payment_twikey/__init__.py

```diff
@@ -1,11 +1,39 @@
-from . import models
-from . import wizard
-from . import controllers
+import logging
+
+from lxml import html
+
+from odoo import SUPERUSER_ID, api
+
+from odoo.addons.payment import reset_payment_provider, setup_provider
+
+from . import controllers, models, wizard
+
+
+def insert_twikey_url_in_email_template(env):
+    template = env.ref("account.email_template_edi_invoice")
```

Comment:
Dient dit ook te gebeuren voor `email_template_edi_credit_note`? Ik weet niet of Twikey ook credit nota's ondersteunt?

Reply:
Ja, bedankt voor de suggestie. Zit er nu in, zie https://github.com/QuatraInternational/twikey-odoo/compare/8cf75c0637cf2da790cd494c063c06ea6e66b18f..09bad6d77ed6fa57f2f40b21d25fabcffa449887